vignettes/advantages.Rmd

```diff
@@ -0,0 +1,176 @@
+
+Especially the extended log printing of errors on the CI service is a huge advantage - often enough, CI services cut log output of the console early, omitting important information about the error. 
+
+Here is an short excerpt of the improved log output by *rcmdcheck* including an error message:
+
+```{r eval = FALSE}
+R CMD check ──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
+─  using log directory ‘/tmp/RtmpM4zdgT/file12d469173dd1/tic.Rcheck’ (581ms)
+─  using R version 3.5.1 (2018-07-02)
+─  using platform: x86_64-pc-linux-gnu (64-bit)
+─  using session charset: UTF-8
+─  using options ‘--no-manual --as-cran’
+✔  checking for file ‘tic/DESCRIPTION’
+─  checking extension type ... Package
+─  this is package ‘tic’ version ‘0.2.13.9010’
+─  package encoding: UTF-8
+N  checking CRAN incoming feasibility (59.7s)
+   Maintainer: ‘Kirill Müller <krlmlr+r@mailbox.org>’
+   
+   New submission
+   
+   Version contains large components (0.2.13.9010)
+   
+   Unknown, possibly mis-spelled, fields in DESCRIPTION:
+     ‘Remotes’
+   
+   Suggests or Enhances not in mainstream repositories:
+     travis
+   
+✔  checking package namespace information
+E  checking package dependencies (806ms)
+   Package suggested but not available: ‘drat’
+   
+   
+   See
+     ‘/tmp/RtmpM4zdgT/file12d469173dd1/tic.Rcheck/00check.log’
+   for details.
+   
+   The suggested packages are required for a complete check.
+   Checking can be attempted without them by setting the environment
+   variable _R_CHECK_FORCE_SUGGESTS_ to a false value.
+   
+   See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
+   manual.
+── R CMD check results ───────────────────────────────────────────────────────────────────────────────────────────────────────────── tic 0.2.13.9010 ────
+Duration: 1m 1.3s
+
+❯ checking package dependencies ... ERROR
+  Package suggested but not available: ‘drat’
+  
+  The suggested packages are required for a complete check.
+  Checking can be attempted without them by setting the environment
+  variable _R_CHECK_FORCE_SUGGESTS_ to a false value.
+  
+  See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
+  manual.
+```
+
+## Caching of packages
+
+When using *tic* all dependencies (the ones of the package and of also the ones of all stages of the CI build) are installed in the "before_install" stage. 
+This has the advantage that all packages get added to the cache (even if they are just needed for deployment), speeding up subsequent builds substantially.
+
+More information about the whole CI workflow can be found in the [Build lifecyle](build-lifecycle.html) vignette.
+
+## Easier troubleshooting
+
+*tic* comes with the ability to emulate the CI run locally and debug potential problems. 
+This is possible because `tic.R` is a plain R file that will be sourced when calling `tic()`.
```

Comment:
What is `tic()` ?

I think this can be kept very short, just mentioning that we can load the config with `load_from_file()`, and run it with `tic()`, and link to the corresponding vignette.

Reply:
shortened heavily in https://github.com/ropenscilabs/tic/pull/80/commits/ad0716bc125a3133878b73ea64f9417ce1d90fdc.